config/settings.yml

```diff
@@ -22,14 +22,13 @@
     :get_container_images: true
     :store_unused_images: true
 :workers:
-  :worker_base:
-    :event_catcher:
-      :event_catcher_openshift:
-        :poll: 1.seconds
-      :event_catcher_prometheus:
-        :poll: 20.seconds
-    :queue_worker_base:
-        :ems_metrics_collector_worker:
-          :ems_metrics_collector_worker_openshift: {}
-        :ems_refresh_worker:
-          :ems_refresh_worker_openshift: {}
+  :ems_metrics_collector_worker_openshift:
+    :inherits_from: ems_metrics_collector_worker
+  :ems_refresh_worker_openshift:
+    :inherits_from: ems_refresh_worker
+  :event_catcher_openshift:
+    :inherits_from: event_catcher
+    :poll: 1.seconds
+  :event_catcher_prometheus:
```

Comment:
@agrare I think this might be a bug, because this is duplication the same thing as https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/420/files#diff-e769bbb8c1ba3711c5403b424ed9c218ffafba7f1890ee394717196f28ff4540R73

Reply:
Okay the name is a little confusing because it doesn't call out the manager type.  This is used by `ManageIQ::Providers::Kubernetes::MonitoringManager::EventCatcher` and `ManageIQ::Providers::Openshift::MonitoringManager::EventCatcher`

They both override the `self.settings_name` to be `:event_catcher_prometheus`

It definitely doesn't need to be defined in both places, you can drop it from here

Reply:
Ok, I'l do that in a follow up.  I kind of want to keep this PR pure for now.